localeval.js

```diff
@@ -1,6 +1,10 @@
 // Thaddée Tyl <thaddee.tyl@gmail.com>. License: CC-BY v3.
 "use strict";
 var node_js = typeof exports === 'object';
+if ("undefined" !== typeof process && process.browser) {
+    // webpack sets browser property on process global
+    node_js = false;
+}
```

Comment:
It'll probably be fine, but that's a hard line for a human like myself to parse because it is very. Also, using `this` to refer to the global object seems a little strange, since `this` is usually used to refer to the current instance of an object. I prefer using `typeof` to check if something is part of the global scope.

These are just my opinions (since you asked). It's your package :P